modules/utils/monitor_phy_param.py

```diff
@@ -0,0 +1,52 @@
+import getopt
+import sys
+import datetime
+import subprocess
+from time import sleep
+from threading import Thread
+
+#defult rssi monitoring interval: 5sec
+rssi_mon_interval = 5
+#defualt interface
+interface = "wlan0"
```

Comment:
For later, we need to think of a smart way to determine the interface. 
In some scripts I've used netifaces (https://pypi.org/project/netifaces/)
But also we need some logic to choose it

Reply:
Yes, agreed.
As of now idea is to get this via args(-i IFNAME) as we can use this in various topology (mesh0/wlan0/p2p0/ibss0..etc..)

Reply:
I put interface names into the conf file:

https://github.com/tiiuae/mesh_com/blob/develop/modules/sc-mesh-secure-deployment/src/mesh_com.conf

Can we pull from there?

Reply:
yeah, sure, it makes sense.
I will add new patchset to retrieve ifname from conf.

Reply:
Sorry, I actually meant from here. I completely forgot to push this.

https://github.com/tiiuae/mesh_com/blob/hotfix/fix_interface_list/modules/sc-mesh-secure-deployment/src/mesh_com.conf

Reply:
Created separate conf for phy param along with interface name.
As of now this module is just for experimental purpose(to study some ongoing research projects) , we can converge to main conf once we finalize on the scheme/radio/topology.